modules/015-admission-policy-engine/crds/doc-ru-security-policy.yaml

```
@@ -66,116 +66,117 @@ spec:
+                      description: Задает доступные значения `runAsGroup`.
                       properties:
                         rule:
-                          description: "Правило определения доступных значений `runAsGroup`."
+                          description: Правило определения значений `runAsGroup`.
                         ranges:
-                          description: "Список диапазонов ID для `runAsGroup`, доступных к использованию в режиме `MustRunAs`."
+                          description: Список диапазонов идентификаторов для `runAsGroup`, доступных в режиме `MustRunAs`.
                           items:
                             properties:
                               min:
-                                description: "Минимальное значение диапазона."
+                                description: Минимальное значение диапазона.
                               max:
-                                description: "Максимальное значение диапазона."
+                                description: Максимальное значение диапазона.
                     supplementalGroups:
-                      description: "Определяет допустимые к использованию значения `supplementalGroups`."
+                      description: Задает доступные значения `supplementalGroups`.
                       properties:
                         rule:
-                          description: "Правило определения доступных значений `supplementalGroups`."
+                          description: Правило определения значений `supplementalGroups`.
                         ranges:
-                          description: "Список диапазонов ID для `supplementalGroups`, доступных к использованию в режиме `MustRunAs`."
+                          description: Список диапазонов идентификаторов для `supplementalGroups`, доступных режиме `MustRunAs`.
                           items:
                             properties:
                               min:
-                                description: "Минимальное значение диапазона."
+                                description: Минимальное значение диапазона.
                               max:
-                                description: "Максимальное значение диапазона."
+                                description: Максимальное значение диапазона.
                     readOnlyRootFilesystem:
-                      description: "Определяет необходимость запуска контейнеров с корневой файловой системой, доступной только для чтения"
+                      description: |
+                      Если установлен в `true`, разрешает запуск только тех подов, у всех контейнеров которых корневая файловая система доступна только для чтения (в поле `securityContext` установлено `readOnlyRootFilesystem: true`). Подробнее в [документации Kubernetes](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#securitycontext-v1-core).
```

```suggestion
                      description: |
                        Если установлен в `true`, разрешает запуск только тех подов, у всех контейнеров которых корневая файловая система доступна только для чтения (в поле `securityContext` установлено `readOnlyRootFilesystem: true`). Подробнее в [документации Kubernetes](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#securitycontext-v1-core).
```

@elenashliaga потерялся отступ